hathor/cli/run_node_args.py

```diff
@@ -64,6 +64,7 @@ class RunNodeArgs(BaseModel, extra=Extra.allow):
     enable_debug_api: bool
     enable_crash_api: bool
     x_sync_bridge: bool
+    x_sync_v1_only: bool
```

Comment:
Why not `sync_v1_only`?

Reply:
Yeah, I think that would actually fit better, I'll change to that.